[YSQL][docs][2024.2.1] YSQL connection manager gflags docs (#25647)
yugabyte/yugabyte-db: docs/content/stable/explore/going-beyond-sql/connection-mgr-ysql.md

| ysql_conn_mgr_server_lifetime | The maximum duration (in seconds) that a backend PostgreSQL connection managed by YSQL Connection Manager can remain open after creation. | 3600 |
| ysql_conn_mgr_log_settings | Comma-separated list of log settings for YSQL Connection Manger. Can include  'log_debug', 'log_config', 'log_session', 'log_query', and 'log_stats'. | "" |
| ysql_conn_mgr_use_auth_backend | Enable the use of the auth-backend for authentication of logical connections. When false, the older auth-passthrough implementation is used. | true |

> | ysql_conn_mgr_readahead_buffer_size | Size of the per-connection buffer(in bytes) used for IO read-ahead operations in YSQL Connection Manager | 8192 |

Reviewer: ```suggestion
| ysql_conn_mgr_readahead_buffer_size | Size of the per-connection buffer (in bytes) used for IO read-ahead operations in YSQL Connection Manager. | 8192 |
```

> | ysql_conn_mgr_tcp_keepalive | TCP keepalive time in YSQL Connection Manager. Set to zero to disable keepalive | 15 |

Reviewer: ```suggestion
| ysql_conn_mgr_tcp_keepalive | TCP keepalive time (in seconds) in YSQL Connection Manager. Set to zero to disable keepalive. | 15 |
```

> | ysql_conn_mgr_tcp_keepalive_keep_interval | TCP keepalive interval(in seconds) in YSQL Connection Manager. This is applicable if 'ysql_conn_mgr_tcp_keepalive' is enabled. | 75 |

Reviewer: ```suggestion
| ysql_conn_mgr_tcp_keepalive_keep_interval | TCP keepalive interval (in seconds) in YSQL Connection Manager. Only applicable if 'ysql_conn_mgr_tcp_keepalive' is enabled. | 75 |
```

> | ysql_conn_mgr_tcp_keepalive_probes | TCP keepalive probes in YSQL Connection Manager. This is applicable if 'ysql_conn_mgr_tcp_keepalive' is enabled. | 9 |

Reviewer: ```suggestion
| ysql_conn_mgr_tcp_keepalive_probes | Number of TCP keepalive probes in YSQL Connection Manager. Only applicable if 'ysql_conn_mgr_tcp_keepalive' is enabled. | 9 |
```

> | ysql_conn_mgr_tcp_keepalive_usr_timeout | TCP user timeout(in milliseconds) in YSQL Connection Manager. This is applicable if 'ysql_conn_mgr_tcp_keepalive' is enabled. | 9 |

Reviewer: ```suggestion
| ysql_conn_mgr_tcp_keepalive_usr_timeout | TCP user timeout (in milliseconds) in YSQL Connection Manager. Only applicable if 'ysql_conn_mgr_tcp_keepalive' is enabled. | 9 |
```

> | ysql_conn_mgr_control_connection_pool_size | Maximum number of concurrent control connections in YSQL Connection Manager. If the value is zero, the default value is 0.1 * ysql_max_connections | 0 |

Reviewer: ```suggestion
| ysql_conn_mgr_control_connection_pool_size | Maximum number of concurrent control connections in YSQL Connection Manager. If set to zero, calculated as 0.1 * ysql_max_connections. | 0 |
```

> | ysql_conn_mgr_pool_timeout | Server pool wait timeout(in milliseconds) in YSQL Connection Manager. Time to wait in milliseconds for an available server. Disconnect client on timeout reach. If the value is set to zero, the client waits for the server connection indefinitely | 0 |

Reviewer: ```suggestion
| ysql_conn_mgr_pool_timeout | Server pool wait timeout (in milliseconds) in YSQL Connection Manager. This is the time clients wait for an available server, after which they are disconnected. If set to zero, clients wait for server connections indefinitely. | 0 |
```

> | ysql_conn_mgr_sequence_support_mode | Sequence support mode when YSQL connection manager is enabled. When set to  'pooled_without_curval_lastval', currval() and lastval() functions are not supported. "When set to 'pooled_with_curval_lastval', currval() and lastval() functions are supported. In these both settings, the monotonic order of sequence is not guaranteed if the "'ysql_sequence_cache_method' is set to 'connection'. To support monotonic order also, set this flag to 'session'. | pooled_without_curval_lastval |

Reviewer: ```suggestion
| ysql_conn_mgr_sequence_support_mode | Sequence support mode when YSQL connection manager is enabled. When set to  'pooled_without_curval_lastval', the currval() and lastval() functions are not supported. When set to 'pooled_with_curval_lastval', the currval() and lastval() functions are supported. For both settings, monotonic sequence order is not guaranteed if `ysql_sequence_cache_method` is set to `connection`. To also support monotonic order, set this flag to `session`. | pooled_without_curval_lastval |
```
